src/js/apps/patients/worklist/filters_app.js

```diff
@@ -19,9 +27,39 @@ export default App.extend({
     this.showFilters();
   },
   showFilters() {
+    this.showAllFiltersButtonView();
     this.showGroupsFilterView();
     this.showOwnerToggleView();
   },
+  showAllFiltersButtonView() {
+    if (this.groups.length < 2) return;
+
+    const ownerView = this.showChildView('allFilters', new AllFiltersButtonView());
+
+    this.listenTo(ownerView, 'click', () => {
+      this.trigger('toggle:filters:sidebar', true);
```

Comment:
I think maybe `toggle:filtersSidebar` might be better.  I _think_ `verb:subject` or `context:verb:subject` is preferable.  I'm not totally convinced, but I think the `:` helps to relevant define parts of the event vs being a word separator.